projects/architecture/interfaces/sidechain-connector.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+import "primitives/heartbeat.proto";
+import "primitives/transactions.proto";
+
+service SidechainConnector {
+  rpc GetHeartbeat (GetHeartbeatInput) returns (GetHeartbeatOutput);
+  rpc CallEthereumContract (CallEthereumContractInput) returns (CallEthereumContractOutput);
+}
+
+message EthereumFunctionParameter {
+    string name = 1;
+    string type = 2;
+}
+
+message EthereumFunctionInterface {
+    string name = 1;
+    repeated EthereumFunctionParameter inputs = 2;
+    repeated EthereumFunctionParameter outputs = 3;
+}
+
+message CallEthereumContractInput {
+  string contractAddress = 1;
+  EthereumFunctionInterface functionInterface = 2;
+  repeated string parameters = 3;
+// string blockNumber = 5;
```

Comment:
Why did you want it? In order to call the contract in a "specific time"?

Reply:
Yes, but it's not a must at the moment